src/main/java/org/broadinstitute/hellbender/tools/walkers/annotator/InbreedingCoeff.java

```diff
@@ -97,4 +115,12 @@ public InbreedingCoeff(final Set<String> founderIds){
     @Override
     public List<String> getKeyNames() { return Collections.singletonList(GATKVCFConstants.INBREEDING_COEFFICIENT_KEY); }
 
-}
+    /**
+     * Entry-point function to initialize the samples database from input data
+     */
+    private Set<String> initializeSampleDB(File pedigreeFile) {
```

Comment:
GIven that you're duplicating a lot of the pedigree-handling code for these two annotations, it might make sense to extract a parent class.  Unless you plan on making more changes when the rest of the arg support goes in.

Reply:
It was a relatively small amount of code but i've gone ahead and refactored it. It is much cleaner now though it might still be subject to change down the road.